torch_xla/core/xla_model.py

```diff
@@ -214,6 +214,17 @@ def xla_replication_devices(local_devices):
   return replication_devices
 
 
+def block_until_ready(tensors):
+  """Blocks the program until `tensors` are materialized.
+    This API is for benchmarking, don't use it in real models.
```

Comment:
If we document it, we automatically open the gates for it to continue to be supported 😉 

It's OK to document it, although I am not entirely sure about the name of the API ... 😄

Reply:
lol I copied the name from JAX :P

Reply:
I feel no matter how `_xla_sync_multi` changes we'll always provide a way to block until the execution is done. So it might be better to have a separate public API for it ;)

Reply:
I vote for `unlazy()` 😄